operations/research-and-development/product/development-process/secure-software-development.md

```diff
@@ -0,0 +1,90 @@
+# Secure Software Development at Mattermost
+
+This document provides guidelines to secure software development at Mattermost. The document is at the moment a [1% draft](/company/about-mattermost/mindsets#drafts-at-1-50-99) and not conclusive.
```

Comment:
Would you say after this it's at 50% or 1% still correct? :-)

Reply:
lol good point i'd say we are at 50% now